kofta/src/app/components/InvitedToJoinRoomModal.tsx

```diff
@@ -66,6 +77,8 @@ export const InvitedToJoinRoomModal: React.FC<Props> = () => {
           <h1 className={`text-2xl mb-2`}>
             {options.type === "someone_you_follow_created_a_room"
               ? t("components.modals.invitedToJoinRoomModal.newRoomCreated")
+              : options.type === "someone_joined_waiting_room"
```

Comment:
nit: Nested ternaries are not a good practice as it is hard to read and looks a bit confusing

Reply:
I understand, I'll replace it with a proper sitch or if statement later. I just wanted to push this feature hit as quickly as I can

Reply:
I'd recommend just making the change while we await review from Ben.